### PR TITLE
Issue:3897880:PDR:Exclusion list tests

### PR DESCRIPTION
**Pull request by vg12345**

### CODE REVIEW

**Comment by vg12345** [comment]
Pylint test integrated and successfully passed at this stage

**Comment by vg12345** [comment]
bot:retest

**Comment by vg12345** [comment]
bot:retest

**Comment by vg12345** [comment]
First successful pytest passed

File: plugins/pdr_deterministic_plugin/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+requests
```
Comment (TalLerner) [inline plugins/pdr_deterministic_plugin/requirements.txt:1]: Did you create this file manually?

Comment (vg12345) [reply plugins/pdr_deterministic_plugin/requirements.txt:1]: Yes. I just looked for dependencies and added them, so the test will finish with successs.

Comment (vg12345) [reply plugins/pdr_deterministic_plugin/requirements.txt:1]: I will generate it automatically

**Pull request closed without merging**